engine/jobmaster/dm/dm_jobmaster.go

```
@@ -212,7 +212,15 @@ func (jm *JobMaster) onWorkerFinished(finishedTaskStatus runtime.FinishedTaskSta
 	taskStatus := finishedTaskStatus.TaskStatus
 
 	finishedStateStore := jm.metadata.FinishedStateStore()
-	err := finishedStateStore.ReadModifyWrite(context.TODO(), func(state *metadata.FinishedState) error {
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+	state, err := jm.metadata.JobStore().Get(ctx)
+	if err != nil {
```

we can just ignore the error